cluster/gce/gci/configure-helper-common.sh

```diff
@@ -617,7 +617,15 @@ function create-master-auth {
     for extra_component in "${extra_components[@]}"; do
       local token="$(secure_random 32)"
       append_or_replace_prefixed_line "${known_tokens_csv}" "${token}," "system:${extra_component},uid:system:${extra_component}"
-      create-kubeconfig "${extra_component}" "${token}"
+      if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
+        if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
+          create-kubeconfig "${extra_component}" "${token}" "${PROXY_RESERVED_IP}" "8888" "http"
+        else
+          create-kubeconfig "${extra_component}" "${token}" "localhost" "8080" "http"
+        fi
```

Comment:
what is the reason If ${KUBERNETES_TENANT_PARTITION} == false, need use 
```
create-kubeconfig "${extra_component}" "${token}" "localhost" "8080" "http"
```
I suggest we use same as 
```
create-kubeconfig "${extra_component}" "${token}"
```

Reply:
Then it will default to secure https port 443, we want to use insecure.

Reply:
as we talked in the meeting, if this is only for POC, that's ok.